### PR TITLE
Update tidb-distributed-execution-framework.md

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tangenta — APPROVED** [review]
rest lgtm

File: tidb-distributed-execution-framework.md [hunk 1]
```diff
@@ -9,11 +9,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 本文档介绍了 TiDB 后端任务分布式框架的使用场景与限制、使用方法和实现原理。
```
Comment (Benjamin2037) [inline tidb-distributed-execution-framework.md:10]:
```suggestion
本文档介绍了 TiDB 分布式框架的使用场景与限制、使用方法和实现原理。
```
要去掉么？

**Review by Benjamin2037 — APPROVED** [review]
rest LGTM

File: tidb-distributed-execution-framework.md [hunk 2]
```diff
@@ -38,9 +34,12 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 - `IMPORT INTO` 用于将 `CSV`、`SQL`、`PARQUET` 等格式的数据导入到一张空表中。详情请参考 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)。
 
+## 使用限制
+- 分布式框架一次只能调度 1 个 `ADD INDEX` 任务，如果第一个 `ADD INDEX` 任务还未完成就提交了第二个 `ADD INDEX` 任务，则第二个 `ADD INDEX` 任务会通过事务的方式来执行。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:38]:
```suggestion
## 使用限制

分布式框架一次只能调度 1 个 `ADD INDEX` 任务。如果第一个 `ADD INDEX` 任务还未完成就提交了第二个 `ADD INDEX` 任务，则第二个 `ADD INDEX` 任务会通过事务的方式来执行。
```

File: tidb-distributed-execution-framework.md [hunk 3]
```diff
@@ -53,7 +52,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 > **注意：**
 >
-> 在升级到 v6.5.0 及以上版本时，建议你检查 TiDB 的 `temp-dir` 路径是否正确挂载了 SSD 磁盘，并确保运行 TiDB 的操作系统用户对该目录有读写权限，否则在运行时可能产生不可预知的问题。该参数是 TiDB 的配置参数，设置后需要重启 TiDB 才能生效。因此，在升级前提前进行设置，可以避免再次重启。
+> 建议你检查 TiDB 的 `temp-dir` 目录至少需要有 100 GiB 的可用空间。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:55]:
```suggestion
> 建议 TiDB 的 `temp-dir` 目录至少有 100 GiB 的可用空间。
```

File: tidb-distributed-execution-framework.md [hunk 4]
```diff
@@ -65,7 +64,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
     在运行后端任务时，框架支持的语句（如 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 和 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)）会采用分布式方式执行。默认集群内部所有节点均会执行后端任务。
 
-2. 根据实际需求，调整可能影响 DDL 任务分布式执行的系统变量：
+2. 根据实际需求，调整可能影响 DDL 任务分布式执行的系统变量，一般情况均使用默认值即可：
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:67]:
```suggestion
2. 一般情况下，对于下列影响 DDL 任务分布式执行的系统变量，使用默认值即可。
```

File: tidb-distributed-execution-framework.md [hunk 5]
```diff
@@ -7,13 +7,9 @@ summary: 了解 TiDB 后端任务分布式框架的使用场景与限制、使
 
 TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个后端任务分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对所有后端任务的统一调度与分布式执行，并为接入的后端任务提供统一的资源管理能力，从整体和单个后端任务两个维度提供资源管理的能力，更好地满足用户对于资源使用的预期。
 
-本文档介绍了 TiDB 后端任务分布式框架的使用场景与限制、使用方法和实现原理。
+本文档介绍了 TiDB 分布式框架的使用场景与限制、使用方法和实现原理。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:10]:
```suggestion
本文档介绍了 TiDB 后端任务分布式框架的使用场景与限制、使用方法和实现原理。
```

File: tidb-distributed-execution-framework.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 > **注意：**
 >
-> 在升级到 v6.5.0 及以上版本时，建议你检查 TiDB 的 `temp-dir` 路径是否正确挂载了 SSD 磁盘，并确保运行 TiDB 的操作系统用户对该目录有读写权限，否则在运行时可能产生不可预知的问题。该参数是 TiDB 的配置参数，设置后需要重启 TiDB 才能生效。因此，在升级前提前进行设置，可以避免再次重启。
+> 建议 TiDB 的 `temp-dir` 目录至少有 100 GiB 的可用空间。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:56]:
```suggestion
> 建议 TiDB 的 [`temp-dir`](/tidb-configuration-file.md#temp-dir-new-in-v630) 目录至少有 100 GiB 的可用空间。
```

**Review by Frank945946 — COMMENTED** [review]
LGTM

File: tidb-distributed-execution-framework.md [hunk 7]
```diff
@@ -27,7 +23,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 - 支持后端任务分布式执行，可以在整个 TiDB 集群可用的计算资源范围内进行灵活的调度，从而更好地利用 TiDB 集群内的计算资源。
 - 提供统一的资源使用和管理能力，从整体和单个后端任务两个维度提供资源管理的能力。
 
-目前，后端任务分布式框架支持分布式执行 `ADD INDEX` 和 `IMPORT INTO`。
+目前，后端任务分布式框架支持分布式执行 `ADD INDEX` 和 `IMPORT INTO` 这两类后端任务。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:26]:
```suggestion
目前，后端任务分布式框架支持分布式执行 `ADD INDEX` 和 `IMPORT INTO` 两类后端任务。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-distributed-execution-framework.md [hunk 8]
```diff
@@ -38,9 +34,13 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 - `IMPORT INTO` 用于将 `CSV`、`SQL`、`PARQUET` 等格式的数据导入到一张空表中。详情请参考 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)。
 
+## 使用限制
+
+分布式框架一次只能调度 1 个 `ADD INDEX` 任务。如果第一个 `ADD INDEX` 任务还未完成就提交了第二个 `ADD INDEX` 任务，则第二个 `ADD INDEX` 任务会通过事务的方式来执行。
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:39]:
```suggestion
分布式框架一次只能调度一个 `ADD INDEX` 任务的分布式执行。如果在当前的 `ADD INDEX` 分布式任务还未执行完成时就提交了一个新的 `ADD INDEX` 任务，则新提交的 `ADD INDEX` 任务会通过事务的方式来执行。
```

File: tidb-distributed-execution-framework.md [hunk 9]
```diff
@@ -65,7 +65,7 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
     在运行后端任务时，框架支持的语句（如 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 和 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)）会采用分布式方式执行。默认集群内部所有节点均会执行后端任务。
 
-2. 根据实际需求，调整可能影响 DDL 任务分布式执行的系统变量：
+2. 一般情况下，对于下列影响 DDL 任务分布式执行的系统变量，使用默认值即可。
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:68]:
```suggestion
2. 一般情况下，对于下列影响 DDL 任务分布式执行的系统变量，使用其默认值即可。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-distributed-execution-framework.md [hunk 10]
```diff
@@ -79,17 +78,13 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
     > - 在分布式任务执行过程中，修改 `tidb_service_scope` 的配置不会对当前任务生效，会从下次任务开始生效。
 
-> **建议：**
->
-> 对于分布式执行 `ADD INDEX` 语句，只需要设置 `tidb_ddl_reorg_worker_cnt`。
-
 ## 实现原理
 
-TiDB 后端任务分布式框架的架构图如下：
+TiDB 后端任务分布式执行框架的架构图如下：
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:83]:
```suggestion
TiDB 分布式执行框架的架构图如下：
```

File: tidb-distributed-execution-framework.md [hunk 11]
```diff
@@ -79,17 +78,13 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
 
-![后端任务分布式框架的架构](/media/dist-task/dist-task-architect.jpg)
+![后端任务分布式执行框架的架构](/media/dist-task/dist-task-architect.jpg)
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:85]:
```suggestion
![分布式执行框架的架构](/media/dist-task/dist-task-architect.jpg)
```

File: tidb-distributed-execution-framework.md [hunk 12]
```diff
@@ -1,33 +1,28 @@
...
-启用 TiDB 后端任务分布式框架能够解决上述问题，并且具有以下三个优势：
+启用 TiDB 分布式执行框架能够解决上述问题，并且具有以下三个优势：
 
 - 提供高扩展性、高可用性和高性能的统一能力支持。
-- 支持后端任务分布式执行，可以在整个 TiDB 集群可用的计算资源范围内进行灵活的调度，从而更好地利用 TiDB 集群内的计算资源。
-- 提供统一的资源使用和管理能力，从整体和单个后端任务两个维度提供资源管理的能力。
+- 支持任务分布式执行，可以在整个 TiDB 集群可用的计算资源范围内进行灵活的调度，从而更好地利用 TiDB 集群内的计算资源。
+- 提供统一的资源使用和管理能力，从整体和单个任务两个维度提供资源管理的能力。
 
-目前，后端任务分布式框架支持分布式执行 `ADD INDEX` 和 `IMPORT INTO`。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:26]:
```suggestion
目前，分布式框架支持分布式执行 `ADD INDEX` 和 `IMPORT INTO`。

```

File: tidb-distributed-execution-framework.md [hunk 13]
```diff
@@ -1,33 +1,28 @@
...
-summary: 了解 TiDB 后端任务分布式框架的使用场景与限制、使用方法和实现原理。
+title: TiDB 分布式执行框架
+summary: 了解 TiDB 分布式执行框架的使用场景、限制、使用方法和实现原理。
 ---
 
-# TiDB 后端任务分布式框架
+# TiDB 分布式执行框架
 
-TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个后端任务分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对所有后端任务的统一调度与分布式执行，并为接入的后端任务提供统一的资源管理能力，从整体和单个后端任务两个维度提供资源管理的能力，更好地满足用户对于资源使用的预期。
+TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对基于该框架执行的任务进行统一调度与分布式执行，并提供整体和单个任务两个维度的资源管理的能力，更好地满足用户对于资源使用的预期。当前该框架支持调度 `ADD INDEX` 和 `IMPORT INTO` 这两类任务。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:8]:
```suggestion
TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对基于该框架执行的任务进行统一调度与分布式执行，并提供整体和单个任务两个维度的资源管理的能力，更好地满足用户对于资源使用的预期。
```

File: tidb-distributed-execution-framework.md [hunk 14]
```diff
@@ -53,46 +53,42 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
-2. 根据实际需求，调整可能影响 DDL 任务分布式执行的系统变量：
+2. 一般情况下，对于下列影响 DDL 任务分布式执行的系统变量，使用其默认值即可。
 
     * [`tidb_ddl_reorg_worker_cnt`](/system-variables.md#tidb_ddl_reorg_worker_cnt)：使用默认值 `4` 即可，建议最大不超过 `16`。
     * [`tidb_ddl_reorg_priority`](/system-variables.md#tidb_ddl_reorg_priority)
     * [`tidb_ddl_error_count_limit`](/system-variables.md#tidb_ddl_error_count_limit)
     * [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size)：使用默认值即可，建议最大不超过 `1024`。
 
-3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
+3. 从 v7.4.0 开始，你可以根据实际需求，调整执行任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行任务。
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:75]:
```suggestion
3. 从 v7.4.0 开始，你可以根据实际需求，调整用于分布式执行框架任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行分布式执行框架的任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行分布式执行框架的任务。如果所有节点均未配置 `tidb_service_scope`，分布式执行框架将调度所有的 TiDB 节点执行任务。
```

File: system-variables.md [hunk 15]
```diff
@@ -4200,12 +4200,12 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- 该变量是一个实例级别的变量，用于控制 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md) 下各 TiDB 节点的服务范围。当设置 TiDB 节点的 `tidb_service_scope` 为 `background` 时，分布式执行框架将调度该节点执行任务（如 [`ADD INDEX`](/sql-statements/sql-statement-add-index.md) 和 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md)）。
 
 > **注意：**
 >
-> - 如果集群内所有节点均未配置 `tidb_service_scope`，所有节点均会执行分布式框架的任务。如果你担心对存量业务有性能影响，可以对其中几个 TiDB 节点设置为 `background`，只有这些节点才会执行分布式框架的任务。
-> - 对于新扩容的节点，如果有运行中的分布式框架的任务还未调度到各个 TiDB 节点，则这个扩容的节点也会执行分布式框架的任务。如果您希望避免占用扩容节点的资源，你需要手动设置所有或者部分存量 TiDB 节点的 `tidb_service_scope` 为 `background`，保证配置为 `background` 的节点执行分布式框架的任务。
+> - 如果集群内所有节点均未配置 `tidb_service_scope`，所有节点均会执行分布式执行框架的任务。如果你担心对存量业务有性能影响，可以对其中几个 TiDB 节点设置为 `background`，只有这些节点才会执行分布式执行框架的任务。
+> - 对于新扩容的节点，如果有运行中的分布式执行框架的任务还未调度到各个 TiDB 节点，则这个扩容的节点也会执行分布式执行框架的任务。如果您希望避免占用扩容节点的资源，你需要手动设置所有或者部分存量 TiDB 节点的 `tidb_service_scope` 为 `background`，保证配置为 `background` 的节点执行分布式执行框架的任务。
```
Comment (qiancai) [inline system-variables.md:4208]:
```suggestion
> - 对于新扩容的节点，如果有运行中的分布式执行框架的任务还未调度到各个 TiDB 节点，则这个扩容的节点也会执行分布式执行框架的任务。如果希望避免占用扩容节点的资源，你需要手动设置所有或者部分存量 TiDB 节点的 `tidb_service_scope` 为 `background`，保证配置为 `background` 的节点执行分布式执行框架的任务。
```

File: tidb-global-sort.md [hunk 16]
```diff
@@ -19,7 +19,7 @@ TiDB 全局排序功能增强了数据导入和 DDL（数据定义语言）操
 
 全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的稳定性与效率。通过将任务需要处理的数据进行全局排序，可以提高数据写入 TiKV 的稳定性、可控性和可扩展性，从而提供更好的数据导入与 DDL 任务的用户体验及更高质量的服务。
 
-全局排序功能在后端任务分布式并行执行框架中执行任务，确保后端任务所需处理的数据在全局范围内保持有序。
+全局排序功能在分布式并行执行框架中执行任务，确保所需处理的数据在全局范围内保持有序。
```
Comment (qiancai) [inline tidb-global-sort.md:22]:
```suggestion
全局排序功能在分布式执行框架中执行任务，确保所需处理的数据在全局范围内保持有序。
```

File: tidb-distributed-execution-framework.md [hunk 17]
```diff
@@ -1,29 +1,29 @@
...
-summary: 了解 TiDB 后端任务分布式框架的使用场景与限制、使用方法和实现原理。
+title: TiDB 分布式执行框架
+summary: 了解 TiDB 分布式执行框架的使用场景、限制、使用方法和实现原理。
 ---
 
-# TiDB 后端任务分布式框架
+# TiDB 分布式执行框架
 
-TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个后端任务分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对所有后端任务的统一调度与分布式执行，并为接入的后端任务提供统一的资源管理能力，从整体和单个后端任务两个维度提供资源管理的能力，更好地满足用户对于资源使用的预期。
+TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是实现对基于该框架执行的任务进行统一调度与分布式执行，并提供整体和单个任务两个维度的资源管理的能力，更好地满足用户对于资源使用的预期。
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:8]:
```suggestion
TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩容能力。从 v7.1.0 开始，TiDB 引入了一个分布式执行框架，以进一步发挥分布式架构的资源优势。该框架的目标是对基于该框架的任务进行统一调度与分布式执行，并提供整体和单个任务两个维度的资源管理能力，更好地满足用户对于资源使用的预期。
```

File: releases/release-7.5.0.md [hunk 18]
```diff
@@ -32,7 +32,7 @@ TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
   <tr>
     <td rowspan="3">稳定性与高可用</td>
     <td>优化<a href="https://docs.pingcap.com/zh/tidb/v7.5/tidb-global-sort" target="_blank">全局排序</a>（实验特性，从 v7.4.0 开始引入）</td>
-    <td>TiDB v7.2.0 中引入了<a href="https://docs.pingcap.com/zh/tidb/v7.5/tidb-distributed-execution-framework" target="_blank">后端任务分布式并行执行框架</a>。在 v7.4.0 中，TiDB 以该框架为基础，引入全局排序，消除了数据 reorg 任务期间临时无序数据导致的不必要的 I/O、CPU 和内存峰值。全局排序利用外部对象存储（目前为 Amazon S3）来存储系统作业期间的中间文件，提高灵活性并降低成本。<code>ADD INDEX</code> 和 <code>IMPORT INTO</code> 等操作将更快速灵活、稳定可靠，且运行成本较低。</td>
+    <td>TiDB v7.2.0 中引入了<a href="https://docs.pingcap.com/zh/tidb/v7.5/tidb-distributed-execution-framework" target="_blank">分布式执行框架</a>。在 v7.4.0 中，TiDB 以该框架为基础，引入全局排序，消除了数据 reorg 任务期间临时无序数据导致的不必要的 I/O、CPU 和内存峰值。全局排序利用外部对象存储（目前为 Amazon S3）来存储系统作业期间的中间文件，提高灵活性并降低成本。<code>ADD INDEX</code> 和 <code>IMPORT INTO</code> 等操作将更快速灵活、稳定可靠，且运行成本较低。</td>
```
Comment (hfxsd) [inline releases/release-7.5.0.md:35]:
```suggestion
    <td>TiDB v7.1.0 中引入了<a href="https://docs.pingcap.com/zh/tidb/v7.5/tidb-distributed-execution-framework" target="_blank">分布式执行框架</a>。在 v7.4.0 中，TiDB 以该框架为基础，引入全局排序，消除了数据 reorg 任务期间临时无序数据导致的不必要的 I/O、CPU 和内存峰值。全局排序利用外部对象存储（目前为 Amazon S3）来存储系统作业期间的中间文件，提高灵活性并降低成本。<code>ADD INDEX</code> 和 <code>IMPORT INTO</code> 等操作将更快速灵活、稳定可靠，且运行成本较低。</td>
```

File: releases/release-7.5.0.md [hunk 19]
```diff
@@ -73,15 +73,15 @@ TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
...
 
     更多信息，请参考[用户文档](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。
 
 ### 性能
 
-* TiDB 后端任务分布式并行执行框架成为正式功能 (GA)，提升并行执行的 `ADD INDEX` 或 `IMPORT INTO` 任务的性能和稳定性 [#45719](https://github.com/pingcap/tidb/issues/45719) @[wjhuang2016](https://github.com/wjhuang2016)
+* TiDB 分布式执行框架成为正式功能 (GA)，提升并行执行的 `ADD INDEX` 或 `IMPORT INTO` 任务的性能和稳定性 [#45719](https://github.com/pingcap/tidb/issues/45719) @[wjhuang2016](https://github.com/wjhuang2016)
 
-    在 v6.6.0 中引入的 TiDB 后端任务分布式并行执行框架成为正式功能 (GA)。TiDB v7.1.0 之前的版本中，在同一时间只有一个 TiDB 节点能够执行 DDL 任务。从 v7.1.0 开始，在分布式并行执行框架下，多个 TiDB 节点可以并行执行同一项 DDL 任务。从 v7.2.0 开始，分布式并行执行框架支持多个 TiDB 节点并行执行同一个 `IMPORT INTO` 任务，从而更好地利用 TiDB 集群的资源，大幅提升 DDL 和 `IMPORT INTO` 任务的性能。此外，你还可以通过增加 TiDB 节点来线性提升 DDL 和 `IMPORT INTO` 任务的性能。
+    在 v6.6.0 中引入的 TiDB 分布式执行框架成为正式功能 (GA)。TiDB v7.1.0 之前的版本中，在同一时间只有一个 TiDB 节点能够执行 DDL 任务。从 v7.1.0 开始，在分布式并行执行框架下，多个 TiDB 节点可以并行执行同一项 DDL 任务。从 v7.2.0 开始，分布式并行执行框架支持多个 TiDB 节点并行执行同一个 `IMPORT INTO` 任务，从而更好地利用 TiDB 集群的资源，大幅提升 DDL 和 `IMPORT INTO` 任务的性能。此外，你还可以通过增加 TiDB 节点来线性提升 DDL 和 `IMPORT INTO` 任务的性能。
```
Comment (hfxsd) [inline releases/release-7.5.0.md:84]:
```suggestion
    在 v7.1.0 中引入的 TiDB 分布式执行框架成为正式功能 (GA)。TiDB v7.1.0 之前的版本中，在同一时间只有一个 TiDB 节点能够执行 DDL 任务。从 v7.1.0 开始，在分布式并行执行框架下，多个 TiDB 节点可以并行执行同一项 DDL 任务。从 v7.2.0 开始，分布式并行执行框架支持多个 TiDB 节点并行执行同一个 `IMPORT INTO` 任务，从而更好地利用 TiDB 集群的资源，大幅提升 DDL 和 `IMPORT INTO` 任务的性能。此外，你还可以通过增加 TiDB 节点来线性提升 DDL 和 `IMPORT INTO` 任务的性能。
```

**Pull request merged**